row_generation/helpers.py

```diff
@@ -1,21 +1,25 @@
+""" Helper functions for the row generation module. """
+
 from typing import List
 
 import itertools
 import re
 
-_cross_pn = []
-_lookup_name = ["Intentionally Left Blank", "1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "E", "T"]
+_CROSS_PN = []
```

Comment:
I think this is cheekily imported in one of the test files

Reply:
Ah yes.  The tests are pretty much completely broken right now because they haven't been edited by any of my refactoring.  But I'm going to go over them next.